cmd/controller-manager/app/options/options.go

```diff
@@ -140,7 +142,14 @@ func (o *Options) AddFlags(flags *pflag.FlagSet, allControllers []string, disabl
 		&o.PrometheusQuantiles,
 		"prometheus-quantiles",
 		map[string]string{"0.5": "0.01", "0.95": "0.01", "0.99": "0.002"},
-		"prometheus summary objective quantiles",
+		"Prometheus summary objective quantiles",
+	)
+
+	flags.StringToStringVar(
```

Comment:
Why not accept a thing here and use [`labels.Parse`](https://pkg.go.dev/k8s.io/apimachinery@v0.28.2/pkg/labels#Parse)?

Reply:
label is a map structure, so it looks like string to string is more appropriate, just like the previous parameter PrometheusQuantiles.